tools/vm-migrator/migrator/subnet_region_migrator.py

```diff
@@ -0,0 +1,507 @@
+#!/usr/bin/env python
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file creates a machine image and them creates an instance
+from the machine image in another subnet
+"""
+
+import argparse
+import logging
+from migrator.exceptions import GCPOperationException
```

Comment:
The linter is failing because `GCPOperationException` is never used in this file.